<launch>

  <!-- Arguments -->

  <param name="/use_sim_time" value="true"/>

  <node name="republish1" type="republish" pkg="image_transport" output="screen" args="compressed in:=/slave1/image_raw raw out:=/slave1/image_raw" />
  <node name="republish2" type="republish" pkg="image_transport" output="screen" args="compressed in:=/slave2/image_raw raw out:=/slave2/image_raw" />

  <arg name="rosbag_filename" default="/home/bjoshi/cave_new.bag"/>
  <arg name="scale" default="1.0"/>
  <arg name="compress_images" default="true"/>

  <node name="sync_images" pkg="utils_ros" type="sync_images" output="screen" > 

    <param name="rosbag_filename" value="$(arg rosbag_filename)"/>
    <param name="scale" value="$(arg scale)"/>
    <param name="compress_image" value="$(arg compress_images)"/>

    <remap from="/camera0" to="/slave1/image_raw"/> 
    <remap from="/camera1" to="/slave2/image_raw"/> 
    <remap from="/imu" to="/imu/imu" />
    <remap from="/sonar" to="/imagenex831l/range"/>

  </node>

</launch>
